nixos/modules/services/backup/borgbackup.nix

```diff
@@ -98,6 +98,34 @@ let
       inherit (cfg) startAt;
     };
 
+  mkCheckService = name: cfg:
+    let
+      userHome = config.users.users.${cfg.user}.home;
+    in nameValuePair "borgbackup-check-${name}" rec {
+      description = "Check BorgBackup repository ${name}";
+      after = [ "borgbackup-job-${name}" ];
+      conflicts = after;
+      path = with pkgs; [
+        borgbackup openssh
+      ];
+      script = ''
+        borg check
+      '';
```

Comment:
Might as well be `ExecStart` for better readability of `systemctl cat` and logs.

Reply:
Correction: no it can't actually! TIL. `ExecStart` looks in the systemd's global PATH (which on NixOS is just `${systemd}/bin`).